include/zephyr/bluetooth/mesh/brg_cfg_srv.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *  @brief Bluetooth Mesh Bridge Configuration Server Model APIs.
+ */
+#ifndef BT_MESH_BRG_CFG_SRV_H__
```

Comment:
Please follow the common pattern in naming

Reply:
Please take a look at the other public headers

Reply:
Will change it to the ZEPHYR_INCLUDE_BLUETOOTH_MESH_, but it is used different things in the other public files. So it is not clear what it should be. But I guess that ZEPHYR... is correct.